vg/draw/canvas.go

```diff
@@ -32,6 +32,23 @@ type TextStyle struct {
 
 	// Font is the font description.
 	Font vg.Font
+
+	// Rotation is the text rotation in radians, counterclockwise from
+	// the default drawing direction.
+	Rotation float64
+
+	// XAlign and YAlign specify the alignment of the text.
+	// They are multiplied by the width
+	// and height of each label respectively and the
+	// added to the final location.  E.g., XAlign=-0.5
```

Comment:
As mentioned above, I changed this to use constants. Let me know what you think.